src/Features/ui/FormOver/FormOver.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  CloseButton,
+  CustomSelectComponent,
+  Form,
+  FormBox,
+  Title,
+} from '@/Shared'
+import {createPortal} from 'react-dom'
+import {useTranslation} from 'react-i18next'
+import CloseIcon from '@mui/icons-material/Close'
+import {ButtonContainer, Overlay} from './StyledComponents'
+import StyledButtonWrapper from './StyledButtonWrapper'
+import {FC, useState} from 'react'
+import {SelectChangeEvent} from '@mui/material'
+
+interface FormOverProps {
+  onClose: () => void
+  title: string
+  mutateFunc: (skill: string, skillMaster: string) => void
+  firstSelectValue: string
+  firstSelectOptions: TransformedArray[]
+  firstSelectTitle: string
+  secondSelectValue: string
+  secondSelectOptions: {value: string; label: string}[]
+  secondSelectTitle: string
+}
+
+export interface TransformedArray {
+  category?: string
+  id?: string
+  label: string
+  value: string
+}
+
+const FormOver: FC<FormOverProps> = ({
+  onClose,
+  title,
+  mutateFunc,
+  firstSelectValue,
+  firstSelectOptions,
+  firstSelectTitle,
+  secondSelectValue,
+  secondSelectOptions,
+  secondSelectTitle,
+}) => {
+  const {t} = useTranslation()
+  const [firstSelects, setFirstSelects] = useState(firstSelectValue)
+  const [secondSelects, setSecondSelects] = useState(secondSelectValue)
+
+  const handleClose = () => {
+    onClose()
+  }
+
+  const handleChangeFirstSelect = (event: SelectChangeEvent<unknown>) => {
+    const value = event.target.value as string
```

Comment:
Изменить в соответствии с комментом к custom select, также другие места и другие файлы
```suggestion
  const handleChangeFirstSelect = (event: SelectChangeEvent<string>) => {
    const value = event.target.value
```